core/capi/cjml.cpp

```diff
@@ -0,0 +1,77 @@
+#include <jml.h>
+
+#include <vector>
+
+#include <jml/math/activation_functions.hpp>
+#include <jml/math/vector.hpp>
+
+// This is the global vector of all JML Vector objects.
+std::vector<jml::Vector> vectors;
+
+// Look up a Vector in the array.
+jml::Vector& vector_lookup(unsigned vid) {
```

Comment:
Should we use an underscore suffix to indicate that this is private?
```suggestion
jml::Vector& vector_lookup_(unsigned vid) {
```